locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/BrooklynImageChooser.java

```diff
@@ -128,15 +128,18 @@ public double score(Image img) {
 
     
         // prefer these guys, in stock brooklyn provisioning
-        score += punishmentForOldOsVersions(img, OsFamily.UBUNTU, 11);
-        score += punishmentForOldOsVersions(img, OsFamily.CENTOS, 6);
 
         OperatingSystem os = img.getOperatingSystem();
         if (os!=null) {
             if (os.getFamily()!=null) {
                 // preference for these open, popular OS (but only wrt versions above) 
-                if (os.getFamily().equals(OsFamily.CENTOS)) score += 2;
+                if (os.getFamily().equals(OsFamily.CENTOS)) {
+                    score += punishmentForOldOsVersions(img, OsFamily.CENTOS, 7);
+                    score += 3;
+
+                }
                 else if (os.getFamily().equals(OsFamily.UBUNTU)) {
+                    score += punishmentForOldOsVersions(img, OsFamily.UBUNTU, 11);
```

Comment:
@neykov It later does:
```
                    if ("14.04".equals(os.getVersion())) score += 0.2;
                    else if ("12.04".equals(os.getVersion())) score += 0.1;
```
so is rewarding 12+, even if it's not punishing 11.

I agree that bumping this to 12 would make sense though. If we changed it to 14, we'd end up preferring 12.10 or 13.10 over  12.04.5 (even though the last one is supported until April 2017, while the first two are not supported).

However, I doubt we ever test our blueprints on 12.04!